etl/src/energuide/element.py

```diff
@@ -63,3 +66,15 @@ def tag(self) -> str:
 
     def insert(self, index: int, element: 'Element') -> None:
         self.__node.insert(index, element.__node)
+
+    def get(self, xpath: str, type_: typing.Type[T]) -> T:
+        try:
+            value = self.__node.xpath(xpath)[0]
+        except IndexError as ex:
+            raise EnerguideError(f"Couldn't find element at {xpath} in {self.tag}", ex)
```

Comment:
Should we define a custom `EnerguideError` subtype instead? Where should it live?

Reply:
I'm not sure either way, what's your opinion?

Reply:
I think we should, it will add slightly more context.